components/Prize/PrizeExamples.jsx

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+import React from 'react'
+import PrizeItems from './PrizeItems'
+
+/** Temporarly div Style can be replaced by tool tip componenet **/
+const divStyle = {
```

Comment:
Do u mean css you to simulate toolip background should be extracted to separate css?